libs/layout/src/lib/directives/drag-and-drop/drag-and-drop-host.directive.ts

```diff
@@ -0,0 +1,62 @@
+import { ContentChildren, Directive, ElementRef, Input, QueryList } from '@angular/core';
+
+import { NgxAccessibleDragAndDropAbstractService } from '../../abstracts';
+import { NgxAccessibleDragAndDropItemDirective } from './drag-and-drop-item.directive';
+import { NgxAccessibleDragAndDropContainerDirective } from './drag-and-drop-container.directive';
+
+/**
+ * A directive to handle accessible drag and drop flows. This directive is meant to be placed on the host of the drag and drop container(s).
+ */
+@Directive({
+	selector: '[ngxAccessibleDragAndDropHost]',
+	exportAs: 'ngxAccessibleDragAndDropHost',
+	standalone: true,
+})
+export class NgxAccessibleDragAndDropHostDirective {
+	/**
+	 * A list of all the drag and drop items
+	 */
+	@ContentChildren(NgxAccessibleDragAndDropItemDirective, { descendants: true })
+	items: QueryList<NgxAccessibleDragAndDropItemDirective>;
+
+	/**
+	 * A list of all the drag and drop containers
+	 */
+	@ContentChildren(NgxAccessibleDragAndDropContainerDirective, { descendants: true })
+	containers: QueryList<NgxAccessibleDragAndDropContainerDirective>;
+
+	/**
+	 * An optional description describing how the drag and drop works.
+	 */
+	@Input({ alias: 'ngxAccessibleDragAndDropHostDescription' }) description: string;
+
+	constructor(
+		private readonly dragAndDropService: NgxAccessibleDragAndDropAbstractService,
+		public readonly elementRef: ElementRef<HTMLElement>
+	) {}
+
+	/**
+	 * Mark a specific drag and drop item as active
+	 *
+	 * @param  id - The id of the drag and drop item
+	 */
+	public markAsActive(id: string): void {
+		this.items?.find((item) => item.itemId === id)?.markAsActive();
+	}
+
+	/**
+	 * Returns the container based on the provided index
+	 *
+	 * @param index - The index of the container
+	 */
+	public getContainer(index: number): NgxAccessibleDragAndDropContainerDirective {
+		return this.containers.find((container) => container.index === index);
+	}
+
+	ngAfterViewInit() {
```

Comment:
Member visibility & return type are missing here.

Reply:
Is there an EsLint rule for that somewhere? Might be good to add it to the list ;P